code/modules/hippie/recycling/trapdoor.dm

```diff
@@ -0,0 +1,171 @@
+//Trap Door
+/obj/machinery/disposal/trapdoor
+	name = "trapdoor"
+	desc = "Almost like a door, but on floor."
+	density = FALSE
+	icon = 'icons/hippie/obj/atmospherics/trapdoor.dmi'
+	icon_state = "closed"
+	mode = FALSE
+	layer = 2.01
+	var/id = 1
+	var/auto_close = 1200
+	var/auto_close_on_mob = 100
+	var/sound_open = 'sound/hippie/machines/blast_door.ogg'
+	var/sound_close = 'sound/hippie/machines/blast_door.ogg'
```

Comment:
Please update the sound cache & use the sound cache by using `""` instead of `''`